benchmark/utils/DriverUtils.swift

```diff
@@ -150,75 +152,16 @@ struct TestConfig {
     }
 
     if let x = benchArgs.optionalArgsMap["--delim"] {
-      if x.isEmpty { return .Fail("--delim requires a value") }
+      if x.isEmpty { return .fail("--delim requires a value") }
       delim = x
     }
 
     if let x = benchArgs.optionalArgsMap["--tags"] {
-      if x.isEmpty { return .Fail("--tags requires a value") }
-      if x.contains("cpubench") {
-        tags.insert(BenchmarkCategories.cpubench)
-      }
-      if x.contains("unstable") {
-        tags.insert(BenchmarkCategories.unstable)
-      }
-      if x.contains("validation") {
-        tags.insert(BenchmarkCategories.validation)
-      }
-      if x.contains("api") {
-        tags.insert(BenchmarkCategories.api)
-      }
-      if x.contains("Array") {
-        tags.insert(BenchmarkCategories.Array)
-      }
-      if x.contains("String") {
-        tags.insert(BenchmarkCategories.String)
-      }
-      if x.contains("Dictionary") {
-        tags.insert(BenchmarkCategories.Dictionary)
-      }
-      if x.contains("Codable") {
-        tags.insert(BenchmarkCategories.Codable)
-      }
-      if x.contains("Set") {
-        tags.insert(BenchmarkCategories.Set)
-      }
-      if x.contains("sdk") {
-        tags.insert(BenchmarkCategories.sdk)
-      }
-      if x.contains("runtime") {
-        tags.insert(BenchmarkCategories.runtime)
-      }
-      if x.contains("refcount") {
-        tags.insert(BenchmarkCategories.refcount)
-      }
-      if x.contains("metadata") {
-        tags.insert(BenchmarkCategories.metadata)
-      }
-      if x.contains("abstraction") {
-        tags.insert(BenchmarkCategories.abstraction)
-      }
-      if x.contains("safetychecks") {
-        tags.insert(BenchmarkCategories.safetychecks)
-      }
-      if x.contains("exceptions") {
-        tags.insert(BenchmarkCategories.exceptions)
-      }
-      if x.contains("bridging") {
-        tags.insert(BenchmarkCategories.bridging)
-      }
-      if x.contains("concurrency") {
-        tags.insert(BenchmarkCategories.concurrency)
-      }
-      if x.contains("algorithm") {
-        tags.insert(BenchmarkCategories.algorithm)
-      }
-      if x.contains("miniapplication") {
-        tags.insert(BenchmarkCategories.miniapplication)
-      }
-      if x.contains("regression") {
-        tags.insert(BenchmarkCategories.regression)
+      if x.isEmpty { return .fail("--tags requires a value") }
+      guard let cat = BenchmarkCategory(rawValue: x) else {
```

Comment:
What about having multiple categories as an input to the driver? The current implementation allows having more than one category inside the set.

Reply:
for example, I can pass `api,Array,regression` as tag's value something I can't do anymore unless I am missing something this late at night?

Reply:
That doesn't work today.

Reply:
But you are right, we /should/ support multiple tags. I actually was surprised when I checked that we didn't. The way you can see is that the optional argument/etc is a `[String : String]`

Reply:
That *DID* work before merging the commit.

Reply:
I talked with @shajrawi offline. Turns out we were expecting people to pass tags using a comma delimited list. I will fix this.